k8s/c2d_deployer.Makefile

```diff
@@ -0,0 +1,96 @@
+#
+# Used with app_v2 Makefile for deployer schema v2.
+#
+
+ifndef __C2D_DEPLOYER_MAKEFILE__
+
+__C2D_DEPLOYER_MAKEFILE__:= included
+
+include ../app_v2.Makefile
+
+
+SOURCE_REGISTRY ?= marketplace.gcr.io/google
+
+##### Helper targets #####
+
+app/build:: $(APP_ID) \
+            .build/$(APP_ID)/deployer \
+            .build/images \
+            .build/$(APP_ID)/tester \
+            .build/$(APP_ID)/VERSION
+
+
+.build/images: $(TARGET_IMAGES)
+
+
+.build/$(APP_ID): | .build
+	mkdir -p "$@"
+
+
+.build/$(APP_ID)/deployer: deployer/* \
+                           chart/$(APP_ID)/* \
+                           chart/$(APP_ID)/templates/* \
```

Comment:
What happens with the V2 application doesn't used Helm Chart?

Reply:
Then this application should not include this file. It can use to app_v2.Makefile and define it's own rules.